package.json

```diff
@@ -28,13 +28,12 @@
     "serve:prod": "webpack-dev-server --color --disable-host-check --config webpack.config.prod-dev.js",
     "test": "jest",
     "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
-    "licenseCheck:prepare": "license-tool/build.sh",
-    "licenseCheck:run": "license-tool/run.sh"
+    "test:watch": "jest --watch"
   },
   "dependencies": {
     "@eclipse-che/devworkspace-client": "^0.0.1-1617871695",
     "@eclipse-che/workspace-client": "^0.0.1-1615287725",
+    "@patternfly/patternfly": "~4.84.0",
```

Comment:
This dependency is not necessary so should be removed along with this line:
https://github.com/eclipse-che/che-dashboard/blob/main/src/index.tsx#L18

Reply:
We need it because of this import 
```
import '@patternfly/patternfly/patternfly-addons.css';
```
https://github.com/eclipse-che/che-dashboard/blob/main/src/index.tsx#L19

We need '@patternfly/patternfly' because of import 'patternfly-addons.css'

Am I right?

Reply:
yep, I've pointed the wrong line. 

We needed that import because of not proper usage of the patterfly components. Now we're OK with that, so the import of the css file and the dependency should be deleted.

Reply:
I have fixed it